### DISCUSSION TITLE
How to create progressbar with this tailwind css file upload form

### DISCUSSION THREAD

**Discussion by asksnbb** [tailwindlabs/tailwindcss#3019] [General]
I am trying to use the progress bar in this tailwind CSS file upload form - https://bbbootstrap.com/snippets/tailwind-css-add-document-form-file-upload-input-37652196

**Comment by simonswiss** [comment]
Hey!

What progress bar are you talking about? Are you asking for design inspiration + code example on how you would implement a progress bar when a user has selected a file to upload?
